scripts/report_metabug_progress.js

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+// A script to report on the percentage progress of items
+// under a bugzilla metabug, for a given time period.
+// We'll use it to assess OKRs of the form "reduce backlog
+// under bug X by Y%".
+//
+// Usage:
+//
+//    node ./scripts/report_metabug_progress bugNumber [startDate endDate]
+//
+
+
+var P = require('bluebird')
+var request = P.promisify(require('request'), { multiArgs: true })
+
+var BZ_URL = 'https://bugzilla.mozilla.org/rest'
+var BZ_STATUS_OPEN = [
+  'UNCONFIRMED',
+  'NEW',
+  'ASSIGNED',
+  'REOPENED'
+]
+
+
+function requestJSON(url) {
+  return request(url).then(function (res) {
```

Comment:
Why not `.spread(function (res, body) {`, since seeing `res[0].statusCode` and `JSON.parse(res[1])` is less obvious.